db/migrate/20200409050122_add_listing_category_to_classified_listing.rb

```diff
@@ -0,0 +1,9 @@
+class AddListingCategoryToClassifiedListing < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :classified_listings,
```

Comment:
I suggest adding a foreign key (`foreign_key: true`)

Reply:
Yup, good point, I usually do.

Reply:
Added, thanks for the suggestion.